### ISSUE TITLE
'too few arguments' error from ProTECT

### ISSUE THREAD

**Issue by Jeltje** [BD2KGenomics/protect#55]
Using the `make develop` method on `commit f31ea0a7d1a0f8fdf5ca51664a14d213c113186b`
`ProTECT --config_file my_parameters.yaml`
gets the usage statement, then
`ProTECT: error: too few arguments`

`pip list`
```
bd2k-python-lib (1.13.dev21)
boto (2.38.0)
cgcloud-lib (1.4a1.dev195)
dill (0.2.5)
pip (8.1.2)
protect (2.0, /home/ubuntu/arjun/protect/src)
PyYAML (3.11)
setuptools (12.2)
toil (3.2.0)
```

**Comment by arkal** [comment]
read the manual (the updated one). You ar emissing the job store.

**Comment by Jeltje** [comment]
Excellent point. However:
`ProTECT --config_file my_parameters.yaml /mnt/data/jobstore`

```
usage: ProTECT [-h] (--config_file CONFIG_FILE | --generate_config)
ProTECT: error: unrecognized arguments: /mnt/data/jobstore
```
Also note that the jobstore is not part of the usage statement. Why is the jobstore not in the config file?

**Comment by arkal** [comment]
That was also fixed in a hotfix (#56)
